etc/config-template.toml

```diff
@@ -924,9 +911,6 @@
 ## The default value is set to min(CPU_NUM * 0.75, 32).
 # num-threads = 24
 
-## Number of ranges to backup in one batch.
-# batch = 8
```

Comment:
There is only `batch_size`, but no `batch`
https://github.com/tikv/tikv/blob/master/src/config.rs#L2242

Reply:
Alright, you are probably right. Added it again :)